src/redprl/redprl.grm

```diff
@@ -80,7 +80,8 @@ end
 
 %nonassoc SQUIGGLE
 %right LEFT_ARROW RIGHT_ARROW DOUBLE_PIPE SEMI
-%left FUN_APP AT_SIGN
+%left FUN_APP
+%left AT_SIGN
```

Comment:
Ah! Thanks; I somehow thought that putting them on the same line would give them precedence in the order you wrote them, but clearly that wasn't the case 😄  This actually explains some confusing things that happened to me early on. thanks!